funnel/forms/proposal.py

```diff
@@ -93,7 +93,9 @@ class ProposalLabelAdminForm(forms.Form):
 
 class ProposalTransferForm(forms.Form):
     user = forms.UserSelectField(
-        __("Transfer to"), validators=[forms.validators.DataRequired()]
+        __("Transfer to"),
+        description=__("Transfer this proposal to another speaker"),
+        validators=[forms.validators.DataRequired()],
```

Comment:
Minor style issue: WTForms suggests that label and validators are always the first two parameters. https://wtforms.readthedocs.io/en/stable/fields.html#fields-with-custom-constructors.

Reply:
To me pairing label and description makes more sense, so we could ignore the WTForms recommendation and order them by label, description, filters and validators (since filters are applied before validation).

Reply:
That's when you subclass the field to make a custom field. As we're explicitly passing `validators=`, it should be fine. I like label and description together as well.